src/components/GraphiQL.js

```diff
@@ -575,15 +590,24 @@ export class GraphiQL extends React.Component {
     return;
   }
 
+  _addKeyPressEventListeners = () => {
+    // Add shortcut for running a query.
+    document.addEventListener('keydown', this._keyHandler, true);
+  }
+
+  _removeKeyPressEventListeners = () => {
+      document.removeEventListener('keydown', this._keyHandler, true);
+  }
+
   _keyHandler = event => {
     // "Run Query" event.
     if ((event.metaKey || event.ctrlKey) && event.keyCode === 13) {
       event.preventDefault();
-      this._runQueryAtCursor();
+      this.runQueryAtCursor();
     }
   }
 
-  _runQueryAtCursor() {
+  runQueryAtCursor() {
```

Comment:
I think this function could use comments describing the usage as well as `@public` - look at [1] for the example